application/qml/pages/MainPage.qml

```diff
@@ -132,8 +132,15 @@ Page {
                     }
 
                     MyDetailItem {
+                        property bool connected: (battery.chargerConnected || battery.acConnected)
+                        property string chargerType: {
+                            if      ( battery.chargerConnected && !battery.acConnected) { return qsTr("USB") }
+                            else if (!battery.chargerConnected &&  battery.acConnected) { return qsTr("AC") }
+                            else if ( battery.chargerConnected &&  battery.acConnected) { return qsTr("USB") + "/" + qsTr("AC") }
+                            return "unknown power source"
+                        }
                         label: qsTr("Charger connected:")
-                        value: battery.chargerConnected ? qsTr("yes") : qsTr("no")
+                        value: connected ? (qsTr("yes") + " (" + chargerType + ")") : qsTr("no")
```

Comment:
This is pretty much as clean as constructing a multi-part translated string can get :) The string `unknown power source` is untranslated here, but I'll merge this as-is and fix it later, so you can continue with the other PR with least amount of hassle.